src/main/cpp/profiler/spark_rapids_profile_converter.cpp

```diff
@@ -740,6 +1134,24 @@ int main(int argc, char* argv[])
       } else {
         convert_to_nvtxt(in, std::cout, opts);
       }
+    } else if (opts.nvtxw) {
+      if (opts.output_path) {
+        void * nvtxwModuleHandle = nullptr;
+        nvtxwInterfaceCore_t *nvtxwInterface = nullptr;
+        nvtxwSessionHandle_t session;
+        nvtxwStreamHandle_t stream;
+        errorCode = initialize_nvtxw(in, opts.output_path.value().stem(), nvtxwModuleHandle, nvtxwInterface, session, stream);
```

Comment:
Calling `.stem()` here silently truncates the user's intended path and, surprisingly, emits the file in the current directory rather than where the user asked it to be placed.